crates/oxc_language_server/src/main.rs

```diff
@@ -487,26 +552,40 @@ impl Backend {
         let Ok(root_path) = uri.to_file_path() else {
             return None;
         };
-        let mut config_path = None;
-        let config = root_path.join(self.options.lock().await.get_config_path().unwrap());
-        if config.exists() {
-            config_path = Some(config);
-        }
-        if let Some(config_path) = config_path {
-            let mut linter = self.server_linter.write().await;
-            let config = Oxlintrc::from_file(&config_path)
-                .expect("should have initialized linter with new options");
-            let config_store = ConfigStoreBuilder::from_oxlintrc(true, config.clone())
-                .expect("failed to build config")
-                .build()
-                .expect("failed to build config");
-            *linter = ServerLinter::new_with_linter(
-                Linter::new(LintOptions::default(), config_store).with_fix(FixKind::SafeFix),
-            );
-            return Some(config);
+        let oxlintrc;
+        let relative_config_path = self.options.lock().await.get_config_path();
+        if let Some(relative_config_path) = relative_config_path {
+            let config = root_path.join(relative_config_path);
+            config.try_exists().expect("Invalid config file path");
```

Comment:
I think this is a change behavior, not sure if it needs to be called out or not.

Previous behavior would just return `None` if the config file didn't exist. However, now this expects that if you supply a config file name then it should exist.

If you're using nested configs, you should supply `''`.